requirements.txt

```diff
@@ -8,6 +8,6 @@ packaging~=16.8
 Pygments~=2.1
 PyPrint~=0.2.6
 requests~=2.12
-setuptools>=19.0
-testfixtures~=4.14.3
+setuptools>=38.0.0
```

Comment:
Does a lower setuptools version work too? Would be cool if we don't require such a high version^^

Reply:
Let me check